src/tasks/includes/utilities.js

```diff
@@ -13,14 +15,35 @@ const Promise = require('bluebird');
 const utilities = {
 
   /**
-   * Generic error handler for streams called in `watch` tasks (used by gulp-plumber)
+   * Handles the output for any errors that might have been captured
+   * during the build and zip Gulp tasks.
+   *
+   * @memberof slate-cli.utilities
+   */
+  outputErrors: () => {
+    if (!errors.length) {
+      return;
+    }
+
+    gutil.log(gutil.colors.red(`There were errors during the build:\n`));
+
+    errors.forEach((err) => {
+      gutil.log(gutil.colors.red(err));
+    });
+
+    errors = [];
+  },
+
+  /**
+   * Generic error handler for streams called in `watch` tasks (used by gulp-plumber).
+   * Any error that is thrown inside of a task is added to the errors array.
    *
    * @memberof slate-cli.utilities
    * @param {Error} err
    */
-  errorHandler: function(err) { // eslint-disable-line babel/object-shorthand
+  errorHandler: (err) => {
     gutil.log(gutil.colors.red(err));
-    this.emit('end');
```

Comment:
yeah-- this is probably not the best solution to the problem, but errors do not inherently end piped streams in node.  This can be problematic for gulp knowing when to close a task.  If you remove this line, I would make sure to do some specific testing around what happens when this errorHandler is evoked.

some info here: http://blog.ibangspacebar.com/handling-errors-with-gulp-watch-and-gulp-plumber/ <- this is old info (2014), so there may be a better, general way to do this with error handling on streams.